lib/services/project-changes-service.ts

```diff
@@ -292,7 +308,7 @@ export class ProjectChangesService implements IProjectChangesService {
 				const fullFilePath = path.join(projectDir, path.join(filePath, "package.json"));
 				if (this.$fs.exists(fullFilePath)) {
 					const json = this.$fs.readJson(fullFilePath);
-					if (json["nativescript"] && _.startsWith(file, path.join(filePath, "platforms"))) {
+					if (json.nativescript && json.nativescript.platforms && _.startsWith(file, path.join(filePath, "platforms"))) {
```

Comment:
Is that still necessary after we added the project directory to the map with visited paths?

Reply:
Well not "necessary" but still sounds like the proper way to detect plugins. Can revert the change if you prefer so.